cernopendata/templates/cernopendata_theme/page.html

```diff
@@ -35,4 +38,13 @@
 
 {%- endblock body_inner %}
 
+{%- block javascript %}
+  {{super()}}
+  {% assets "cernopendata_glossary_js" %}<script src="{{ ASSET_URL }}"></script>{% endassets %}
+  <script>
+    $(document).ready(function () {
+      $('body').glossary('/static/test_data/glossary.json');
```

Comment:
Do you want to commit a static file for now, and create a new task on how to populate this file from glossary records? Or do you want to address it as part of this PR?

(E.g. we'd need to create this file for zglossary when the Flask app starts, or have a CLI that would compare this file to records and possibly regenerate it, etc.)

Reply:
I agree this static file should be removed and taken from the one in fixtures as they are identical. For now I just put it out of simplicity so as to test the library. I think I will address the issue as part of this PR, thats why I will let WIP.

Reply:
👍 (And let's recall to create this file once we go into "live instance" work package that won't have full fixtures anymore.)